source/2020-01-10-the-ember-times-issue-130.md

```diff
@@ -85,13 +85,11 @@ READMORE
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-
+## [Octane at EmberConf 2020 📢](https://emberconf.com)
```

Comment:
I added a new line after the header. I think this will help pass Travis CI.
```suggestion
## [Octane at EmberConf 2020 📢](https://emberconf.com)

```